autogen/agenteval/criterion.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+
+class Criterion:
+    """
+    A class that represents a criterion for agent evaluation.
+    """
+
+    def __init__(self, name: str, description: str, accepted_values: [str], sub_criteria=None):
+        """
+        args:
+        - name (str): The name of the criterion.
+        - description (str): The description of the criterion.
+        - accepted_values ([str]): The list of accepted values for the criterion.
```

Comment:
`sub_criteria` is not described in docstring, and the accepted type is undefined.